docs/products/clickhouse/howto/manage-databases-tables.md

```diff
@@ -33,20 +33,59 @@ can create a database through the web interface of the
     necessary customizations and run secondary queries to grant access
     to the admin user.
 
+</TabItem>
+<TabItem value="cli" label="SQL">
+Run the following SQL command:
+
+```sql
+CREATE DATABASE DATABASE_NAME
```

Comment:
Some important facts:
1. For now we will allow creating database only to the default user (avnadmin)
2. User can create only replicated database.
Here are examples with error texts
![image](https://github.com/user-attachments/assets/3bf16c9e-1d13-4981-b39f-ab67240dbf1e)

Reply:
Thank you @Khatskevich ! I assume both those limitations (avnadmin user and Replicated enigne) are also true for db creation in the Aiven Console, correct?

Reply:
Not really. Console has a bit less constraints.
In console everything the interface is those buttons
![image](https://github.com/user-attachments/assets/d34230db-c5fb-4b4f-bc99-0522bfb66bae)

So, privileges to create database can be cranted to many users and along with Replicated databases, some integration databases can be created as well.